package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "asset-manager",
+  "version": "1.0.0",
+  "description": "Manages Evolv distributed assets",
+  "main": "src/index.js",
+  "type": "module",
+  "scripts": {
+    "test": "mocha",
+    "test:debug": "node --inspect node_modules/.bin/mocha",
+    "lint": "eslint src",
+    "build": "rollup --config",
+    "commitmsg": "validate-commit-msg",
+    "destroy": "rm -rf node_modules package-lock.json dist",
+    "deploy": "npm run destroy && npm install && npm run build && aws s3 cp ./dist/webloader.js  s3://evolv-execution-plan-binaries/asset-manager/releases/latest/webloader.js --acl public-read && aws s3 cp ./dist/webloader.min.js  s3://evolv-execution-plan-binaries/asset-manager/releases/latest/webloader.min.js --acl public-read"
```

Comment:
Protip: npm-scripts has a feature in which prepending a script name with `pre` or `post` will instruct NPM before or after the script that has been called.

For example:
```
"predeploy": "npm run destroy && npm install && npm run build",
"deploy": "aws s3 cp"
```
When you call `npm run deploy`, NPM will automatically run `predeploy` before `deploy`.

Reply:
I do think, however, that the `deploy` script should instead be moved in the _.travis.yml_ file.

Reply:
Yah this was only temporary until we get a ci and release process.